runtime/elem/GraphRenderSequence.h

```diff
@@ -178,7 +178,7 @@ namespace elem
         {
             // Don't promote if our RootRenderSequence represents a RootNode that has become
             // inactive, even if it's still fading out
-            if (rootPtr->getTargetGain() < FloatType(0.5))
+            if (!rootPtr->active() && rootPtr->stillRunning())
```

Comment:
Just noticing this– I think this changes the semantics of the conditional. We want to return early if the root node is inactive, even if it's still fading out.

I have a ticket in my todo-list to revisit here to smoothly fade out the feedback taps here, but until we get to that I'd say let's preserve the existing behavior here

Reply:
hmm, maybe there's some semantics i'm not understanding here? i'm assuming that `active` means running and not fading out, and `stillRunning` means either active or inactive but still fading out.